network-api/networkapi/buyersguide/templates/product_page.html

```diff
@@ -11,12 +11,21 @@
 
 {% block guts %}
 <div class="container-fluid text-center product-header bg-gray">
-  <img class="mb-0" src="{{mediaUrl}}{{"AWS_LOCATION"|env}}/{{product.image}}" width="400" alt="{{product.name}}"/>
+  <div class="container">
+    <div class="row">
+      <div class="col-12 col-md-10 offset-md-1">
+        <img class="mb-0" src="{{mediaUrl}}{{"AWS_LOCATION"|env}}/{{product.image}}" width="400" alt="{{product.name}}"/>
+        {% if product.meets_minimum_security_standards %}
```

Comment:
This works, but given that this is a tristate value, it's better to be explicit about which we expect to find:
```suggestion
        {% if product.meets_minimum_security_standards is True %}
```